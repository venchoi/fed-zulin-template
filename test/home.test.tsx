import App from '../src/routers/ui';
import { mount, shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json'; //做快照
test('login test', async () => {
    console.log('App-mountComponent test function  begin ');
    // const wrapper = shallow(<App changeShowContent={() => {}} />);
    // toJson(wrapper, {
    //     noKey: false,
    //     mode: 'deep',
    // });
    // expect(wrapper).toMatchSnapshot();

    console.log('App-mountComponent test function  stop  --success ');
});
