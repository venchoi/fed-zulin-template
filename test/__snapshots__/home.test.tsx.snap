// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login test 1`] = `ShallowWrapper {}`;
